### PR TITLE
feat: add cmd wallet alias

### PR DESCRIPTION
**Pull request by vhbien000**
**What does this PR do?**

-   [x] add cmd wallet alias

**How to test**
- `/wallet alias set <address/current alias> <new alias>` ->  add/update alias
image

- `/wallet alias remove <address/current alias>` -> remove alias
image

- Error messages
image

- Reformat wallet watchlist, show both address and alias
image

### CODE REVIEW

File: src/commands/wallet/alias/set/processor.ts [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  msgColors,
+  shortenHashOrAddress,
+} from "utils/common"
+import { somethingWentWrongPayload } from "utils/error"
+
+export async function updateAlias(
+  msg: OriginalMessage,
+  author: User,
+  address: string,
+  alias = ""
```
Comment (anhnh12) [inline src/commands/wallet/alias/set/processor.ts:21]: alias length limit?

File: src/commands/wallet/alias/set/processor.ts [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+  if (status === 409) {
+    throw new InternalError({
+      msgOrInteraction: msg,
+      title: "Alias has been used",
+      description: `\n${pointingright} This alias has been used for another address. Please enter another alias!\n${pointingright} You can see used aliases by using ${await getSlashCommand(
+        "wallet list"
+      )}.`,
+    })
+  } else if (status == 404) {
```
Comment (anhnh12) [inline src/commands/wallet/alias/set/processor.ts:43]: ===

File: src/commands/wallet/alias/set/processor.ts [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+    throw new InternalError({
+      msgOrInteraction: msg,
+      title: "Wallet not found",
+      description: `\n${pointingright} Can't find any wallet from ${
+        valid
+          ? "address " + "`" + shortenHashOrAddress(address) + "`"
+          : "alias " + "`" + address + "`"
+      }. Please try again later.`,
```
Comment (anhnh12) [inline src/commands/wallet/alias/set/processor.ts:51]:
```suggestion
        valid
          ? `${address} \`${shortenHashOrAddress(address)}\``
          : `${alias} \`${address}\``
      }. Please try again later.`,
```

File: src/commands/wallet/alias/set/processor.ts [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+            "wallet alias"
+          )} to change to another alias.\n
+**Detail**
+${getEmoji("WALLET_1")} \`Wallet.    ${shortenHashOrAddress(
+            data?.address
+          )}\` ${getEmoji(
+            AddressChainType[chainType as keyof typeof AddressChainType]
+          )}
+${getEmoji("CONVERSION")} \`Alias.     ${
+            alias.slice(0, 16) + (alias.slice(0, 16).length == 16 ? "..." : "")
```
Comment (anhnh12) [inline src/commands/wallet/alias/set/processor.ts:92]: try func shortenHashOrAddress()

File: src/commands/wallet/alias/set/processor.ts [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+            "wallet list"
+          )}.
+${pointingright} Use ${await getSlashCommand(
+            "wallet alias"
+          )} to change to another alias.\n
+**Detail**
+${getEmoji("WALLET_1")} \`Wallet.    ${shortenHashOrAddress(
+            data?.address
+          )}\` ${getEmoji(
+            AddressChainType[chainType as keyof typeof AddressChainType]
```
Comment (anhnh12) [inline src/commands/wallet/alias/set/processor.ts:123]:
```suggestion
                        getEmoji(chainType as EmojiKey)
```

File: src/commands/wallet/alias/set/processor.ts (hunk 1 above)
Comment (vhbien000) [reply src/commands/wallet/alias/set/processor.ts:21]: just only trim when length >= 16 now

**Review by anhnh12 — APPROVED** [review]

**Pull request merged**